cli/src/main/java/com/devonfw/tools/ide/cli/Ide.java

```diff
@@ -179,7 +179,7 @@ private void initializeJlineCompletion() {
           } catch (EndOfFileException e) {
             // CTRL+D
             return;
-          } catch (CliException e) {
+          } catch (Exception e) {
             context.error("An error occurred while using autocompletion: {}", e);
```

Comment:
You could maybe split the try block. So you can catch auto completions error first, then have a different try block for `runCommand()`

Reply:
Seems to be addressed. @MattesMrzik can we resolve this?